elrond-wasm/src/types/managed/basic/big_uint.rs

```diff
@@ -110,7 +136,7 @@ impl<M: ManagedTypeApi> BigUint<M> {
     #[inline]
     pub fn zero() -> Self {
         let handle: M::BigIntHandle = M::static_var_api_impl().next_handle();
-        M::managed_type_impl().bi_set_int64(handle.clone(), 0);
+        Self::set_value(handle.clone(), 0u32);
```

Comment:
Same as for BigInt, no need to go through all the checks

Reply:
Done.